src/main/java/MovieInformation/controller/MovieInformationController.java

```diff
@@ -0,0 +1,42 @@
+package MovieInformation.controller;
+
+import MovieInformation.MovieInformationNotFoundException;
+import MovieInformation.entity.Movie;
+import MovieInformation.service.MovieInformationService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class MovieInformationController {
+    private final MovieInformationService movieInformationService;
+
+    public MovieInformationController(MovieInformationService movieInformationService) {
+        this.movieInformationService = movieInformationService;
+    }
+
+    //GETの実装
+    //全件取得の実装
+    @GetMapping("/movie-information")
+    public List<Movie> getAllMovieInformation() {
+        List<Movie> movies = movieInformationService.getAllMovieInformation();
+        return movies;
```

Comment:
IntelliJでこのように書いては？というレコメンドが出ないでしょうか？？
```suggestion
        return movieInformationService.getAllMovieInformation();
```
黄色い波線が出ていたら、マウスオーバーしておすすめされてる修正方法をクリックして、自動的にIntelliJに修正してもらいましょう！

Reply:
266da96f2fd4440edf2f715d1999c32a7b1042b3
こちら可能な限り、修正しました！